core/templates/dev/head/tests/form_builder_test_page.html

```diff
@@ -136,7 +136,7 @@ <h5><[form.name]></h5>
   <script src="/templates/dev/head/filters/WrapTextWithEllipsisFilter.js"></script>
   <script src="/templates/dev/head/tests/FormBuilderTestPage.js"></script>
   <script src="/templates/dev/head/tests/FormOverlayDirective.js"></script>
-  <script src="/templates/dev/head/components/CkEditorRteDirective.js"></script>
+  <script src="/templates/dev/head/components/ck-editor-helpers/ck-editor-4.directive.ts"></script>
```

Comment:
Hmm, can you please explain this change? From a renaming point of view, it should be something like `CkEditor4RteDirective.js`.

Reply:
Hi @apb7,  There is no such file `CkEditorRteDirective.js` i think  @YashJipkate missed this?

Reply:
These files are not served and are not used in automated tests and hence these got missed. They are not even part of the TS or Webpack migration it seems and it is being ignored since then.

Reply:
Just a general note @YashJipkate, make sure to not rely only on the automated tests to catch things -- they help, but should not be used as a full source of truth, especially since the automated tests for the frontend are incomplete. You should always use grep, searching, manual testing, etc. to find all the cases that are needed.